Foundation/Process.swift

```diff
@@ -445,6 +445,12 @@ open class Process: NSObject {
           environment["PWD"] = currentDirectoryURL.path
         }
 
+        // On Windows, the PATH is required in order to locate dlls needed by
+        // the process so we should also pass that to the child
+        if environment["Path"] == nil, let path = ProcessInfo.processInfo.environment["Path"] {
```

Comment:
Is the variable not `PATH` rather than `Path`?

Reply:
It is (surprisingly) actually `Path`. cmd.exe/powershell are case insensitive to it so `PATH` works, but the actual environment variable is `Path` if you print it out/open the environment variable editor.

Reply:
*blink* I hate windows.